packages/atomic/src/components/atomic-result-list/atomic-result-list.tsx

```diff
@@ -117,296 +117,53 @@ export class AtomicResultList implements InitializableComponent {
   @Method() public async setRenderFunction(
     render: (result: Result) => HTMLElement
```

Comment:
```suggestion
    render: RenderingFunc
```

Reply:
sooo my idea here was to make it clear what kind of result you are receiving (if you are in `AtomicResultList`, you will only get `Result`, so you need to handle that in your rendering function. If you are in the folded one, you only get `FoldedResult` and therefore can render children, etc